content/guides/getting-started.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Getting Started
+sort: 2
+contributors:
+  - bebraw
+  - varunjayaraman
+  - cntanglijun
+  - chrisVillanueva
+  - johnstew
+  - simon04
+  - aaronang
+  - jecoopr
+---
+
+As you may already know, webpack is used to compile JavaScript modules. Once [installed](/guides/installation), you can interface with webpack either from its [CLI](/api/cli) or [API](/api/node). If you're still new to webpack, please read through the [core concepts](/concepts) and [this comparison](/guides/comparison) to learn why you might use it over the other tools that are out in the community.
+
+
+## Basic Setup
+
+First let's create a directory, initialize NPM, and [install webpack locally](/guides/installation#local-installation):
+
+``` bash
+mkdir webpack-demo && cd webpack-demo
+npm init -y
+npm install --save-dev webpack
+```
+
+Now we'll create the following directory structure and contents:
+
+``` diff
+webpack-demo
+|- package.json
++ |- index.html
++ |- /src
++   |- index.js
+```
+
+__src/index.js__
+
+```javascript
+function component () {
+  var element = document.createElement('div');
+
+  /* lodash is required for the next line to work */
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+
+  return element;
+}
+
+document.body.appendChild(component());
+```
+
+__index.html__
+
+```html
+<html>
+  <head>
+    <title>webpack 2 demo</title>
+    <script src="https://unpkg.com/lodash@4.16.6"></script>
+  </head>
+  <body>
+    <script src="/src/index.js"></script>
+  </body>
+</html>
+```
+
+In this example, there are implicit dependencies between the `<script>` tags. Our `index.js` file depends on `lodash` being included in the page before it runs. This is because `index.js` never declared a need for `lodash`; it just assumes that the global variable `_` exists.
+
+There are problems with managing JavaScript projects this way:
+
+- It is not immediately apparent that the script depends on an external library.
+- If a dependency is missing, or included in the wrong order, the application will not function properly.
+- If a dependency is included but not used, the browser will be forced to download unnecessary code.
+
+Let's try using webpack to manage these scripts instead...
+
+
+## Creating a Bundle
+
+First we'll tweak our directory structure slightly, separating the "source" code we write and edit (`/src`) from our "distribution" code (`/dist`) which is actually shipped to the browser:
+
+``` diff
+webpack-demo
+|- package.json
++ |- dist
++   |- index.html
+- |- index.html
+|- /src
+  |- index.js
+```
+
+To bundle the `lodash` dependency with `index.js`, we'll need to install the library locally...
+
+```bash
+npm install --save lodash
+```
+
+and then import it from our script...
+
+__src/index.js__
+
+```diff
++ import _ from 'lodash';
+
+function component () {
+// ...
+```
+
+We also need to change our update our `<script>` tags to point to the unified bundle:
```

Comment:
Suggestion: "Since we now start bundling our scripts, we need to update our `<script>` tag to load the bundle, instead of individual files. We should also remove the `<script>` tag that loads lodash, as we now import lodash in our script with webpack:"